src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuefinaldecision/IssueFinalDecisionAboutToSubmitHandler.java

```diff
@@ -31,16 +43,22 @@
 @Slf4j
 public class IssueFinalDecisionAboutToSubmitHandler implements PreSubmitCallbackHandler<SscsCaseData> {
 
+    @Value("${feature.snl.enabled}")
+    private boolean scheduleListingEnabled;
```

Comment:
I prefer to include this within the constructor to make it easier from unit tests.